packages/studio-ui/src/store/models/StudioStore.ts

```diff
@@ -22,4 +23,5 @@ export type StudioStore = {
   actions: StudioActions;
   studioConfig: StudioConfigSlice;
   studioEnvData: StudioEnvDataSlice;
+  gitData?: GitData;
```

Comment:
is this something we still want to add right now, even if we don't do anything with it or have a slice for it yet?

Reply:
Oh oop sorry I'll remove this -